### PR TITLE
add property rows to textarea

### PR DESCRIPTION
**Pull request by orangeven**
add property rows to textarea

### CODE REVIEW

File: bundle/src/main/webapp/app-root/clientlibs/io.wcm.caconfig.editor/js/widgets/property-input-textarea.directive.js [hunk 1]
```diff
@@ -45,6 +45,8 @@
       var input = inputMap[scope.property.metadata.type];
       scope.pattern = input.pattern;
       scope.i18n = $rootScope.i18n;
+
+      scope.textareaRows = props.textareaRows;
```
Comment (stefanseifert) [inline bundle/src/main/webapp/app-root/clientlibs/io.wcm.caconfig.editor/js/widgets/property-input-textarea.directive.js:49]: throws: _ReferenceError: props is not defined_

**Review by stefanseifert — CHANGES_REQUESTED** [review]
please also update the changelog

File: bundle/src/main/resources/angularjs-partials/propertyInputTextarea.html [hunk 2]
```diff
@@ -3,6 +3,7 @@
     <textarea is="coral-textarea"
               ng-model="property.value" ng-trim="false"
               ng-required="{{property.metadata.properties.required && !property.overridden && !property.inherited}}"
+              rows="{{textareaRows}}"
```
[the remark this answers is not in the document]
Comment (orangeven) [reply bundle/src/main/resources/angularjs-partials/propertyInputTextarea.html]: I tried `ng-attr-rows="{{textareaRows || undefined }}"`. rows attribute is now skipped as expected if textareaRows is not defined. but `ng-attr-rows="{{textareaRows || undefined }}"` is already rendered in the html markup. is it the expected hehavior?

[the remark this answers is not in the document]
Comment (stefanseifert) [reply bundle/src/main/resources/angularjs-partials/propertyInputTextarea.html]: you're right - ng-attr-rows seems to be always present. i've reverted to the your previous version, the property "rows" is not present if textareaRows is not set.

**Review by stefanseifert — APPROVED** [review]

**Pull request merged**